apps/tng-radio-hub/src/Components/pages/contact/contact.tsx

```diff
@@ -0,0 +1,26 @@
+import './contact.module.css';
+
+/* eslint-disable-next-line */
+export interface ContactProps {}
+
+export function Contact() {
+    return (
+        <div className="contact">
+      <div className="backDrop">
+        <h1>Contacts</h1>
+        <h2>Racine Jeff</h2>
```

Comment:
You should create a data structure for employees.

```ts
type RadioEmployee = {
    fullname: string;

    position: string;
}

const allEmployees: RadioEmployee[] = [
{
    fullname: 'Braeden Boyd',
    position: 'Digitization Technician'
}
]
```

Then loop over this data structure to create one (probably a Card- ask Justin) block of html for each employee. Ideally, you will break this out into a presentational component (widget) named something like `EmployeeInfo` that takes the `RadioEmployee` type in its props and whose only job is to present said data.

It's really important that you only write the 
```html
<p>{title}</p>
<h2>{fullname}</h2>
```
block once so that you have a single source of truth for this and so that you can easily change it for all employees in one go. Imagine trying to maintain this code when you have 100 employees and you want to register a new property or use a different html element to wrap one of the props.